docs/blog/november-contributions-showcase.md

```diff
@@ -0,0 +1,111 @@
+
+Really great work here. This helped us reduce bundle sizes in a whole bunch of projects.
+
+### sonda
+
+As part of the cleanup effort in the e18e space, it is a common need that we want to analyze the bundle of a project. Usually, we do this to find duplicate dependencies, oversized dependencies and so on.
+
+In the wild, there are not many tools to do this analysis and do it accurately. This has left us with often manually inspecting bundles and the process taking a lot longer than it should.
+
+All good, though, [Filip](https://bsky.app/profile/filipsobol.bsky.social) has come to the rescue with his new tool: [sonda](https://github.com/filipsobol/sonda/tree/main/packages/sonda)!
+
+Sonda is a bundle analyzer which works with all popular bundlers by leaning on sourcemaps to determine the bundle contents. This works great and has already helped us with many investigations into footprint reduction of some projects :pray:
+
+## Contributions / Improvements
+
+### bluesky
+
+Some fun collaboration kicked off in the [e18e discord](https://chat.e18e.dev) recently. A few members of the community started looking into the bluesky bundle size, and noticed a few inefficiencies :eyes:
+
+While digging around, they noticed [zod](https://github.com/colinhacks/zod) was included twice in the bundle!
+
+A simple lockfile update allowed this to be de-duped, and shaved off a fair chunk of the bundle.
+
+Super work here by [@marvinh.dev](https://bsky.app/profile/marvinh.dev) and [@jviide.iki.fi](https://bsky.app/profile/jviide.iki.fi) :tada:
+
+You can read more about the investigation and the solution in the [pull request](https://github.com/bluesky-social/social-app/pull/6610).
+
+## Other news
+
+### deoptexplorer
+
+More of a "you should know about this" than news - [deoptexplorer](https://github.com/microsoft/deoptexplorer-vscode) is an extremely useful VSCode plugin to help you find deoptimisations in your code.
+
+In the community, we have started using it heavily to find performance bottlenecks in many different popular tools.
+
+Big thanks to microsoft for building this! Check it out and maybe you'll find some performance improvements :rocket:
+
+### `require(esm)` unflagged in Node 22.x
+
+In Node 22.x, the ability to `require(...)` ES modules has been unflagged and enabled by default!
+
+This is huge news and should open many doors for us being able to migrate libraries to ESM-only. We should also be able to gradually start removing dual-package publishing from various popular packages and go all in on ESM.
+
+Read more about this in [Node's blog post](https://nodejs.org/en/blog/release/v22.12.0#requireesm-is-now-enabled-by-default).
+
+Excellent work by [@joyeecheung.bsky.social](https://bsky.app/profile/joyeecheung.bsky.social) as always :pray:
+
+### vite 6.0
+
+TODO
+
+ref: https://github.com/vitejs/vite/blob/main/packages/vite/CHANGELOG.md#performance
+
```

Comment:
there's also the switch to tinyglobby and chokidar 4, which is up above in the breaking changes section, and which reduce bundle size